src/main/java/org/mule/extensions/jms/internal/connection/provider/GenericConnectionProvider.java

```diff
@@ -93,4 +122,59 @@ public JmsSupport create102bSupport() {
       }
     };
   }
+
+  protected void configureSSLContextIfNeeded() {
```

Comment:
Due to there being another method to configureSSLContext, maybe it is better to change this name to something like addCertificatesToSSLConetextIfNeeded or something that avoids confusion.

Reply:
method was renamed.